src/js/API/peaks/methods.ts

```diff
@@ -43,3 +49,31 @@ export const completeThePeak = async (
     return undefined;
   }
 };
+
+export const getFirstConqueror = async (
```

Comment:
Z tego co rozumiem to `getFirstConqueror` jak i `getPeakTotalCompletions` mają być wykorzystane tylko i wyłącznie w statystykach szczytu. W takim razie nie widzę większego sensu rozdzielania tego na osobne funkcje, raczej zwróciłbym cały obiekt modelu PeakStatistics.

Reply:
Nawet lepiej byłoby to połączyć w jeden endpoint po stronie backu, który zwracałby całe statystyki dla szczytu

Reply:
Też o tym myślałam, na razie zrobiłam tak jak jest na backu.

Reply:
Ok to możemy w takim razie to tu tak zostawić i dorzucić TODO, do zrobienia po zmianach na backu